src/styles/components/_c-landing.scss

```diff
@@ -19,41 +26,39 @@
       bottom: 0;
     }
     .phone-image {
```

Comment:
About this styles, maybe a solution for the resizing issue can be to set the screenshot image as a child of the phone image and make the position of it relative to its parent.